app/Nova/User.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+namespace App\Nova;
+
+use Laravel\Nova\Fields\ID;
+use Illuminate\Http\Request;
+use Laravel\Nova\Fields\Text;
+use Laravel\Nova\Fields\Gravatar;
+use Laravel\Nova\Fields\Password;
+
+/**
+ * @codeCoverageIgnore
```

Comment:
I think it would be more suitable to exclude the `/Nova/` folder recursively using the `codecov.yaml` file. Ref: https://docs.codecov.io/v4.3.6/docs/ignoring-paths

Reply:
I forgot you used this. I added these tags to exclude from code coverage in php unit. Will address tomorrow

Reply:
Ah, makes sense